packages/bbui/src/Tabs/Tabs.svelte

```diff
@@ -57,10 +57,8 @@
   function calculateIndicatorLength() {
     if (!vertical) {
       width = $tab.info?.width + "px"
-      height = $tab.info?.height
     } else {
       height = $tab.info?.height + 4 + "px"
-      width = $tab.info?.width
```

Comment:
We only need one dimension of sizing for each tab layout. Adding these other sizes caused an issue with tab heights in dev tools.